src/lib/components/ProjectsPage/ProjectsTable.tsx

```diff
@@ -56,9 +58,22 @@ export const ProjectsTable = ({ projectType }: IProjectPageProps) => {
     setActiveFilters({ ...activeFilterFields });
   }, [watchedFields]);
 
+  const filterBySearch = (row: any) => {
+    if (searchValue.length) {
```

Comment:
add trim() before... so filter by empty string won't affect this

Reply:
done!